app/assets/javascripts/spree/backend/translations.js.coffee

```diff
@@ -30,7 +30,7 @@ $ ->
     display_locale_fields()
     false
 
-  $('#locale, #supported_locales_').select2({placeholder: Spree.translations['please_choose_language']})
+  $('#locale, #available_locales_').select2({placeholder: Spree.translations['please_choose_language']})
```

Comment:
@tvdeyen, 
Do I understand it correctly that this page - http://localhost:3000/admin/locale - 
supposed to have 2 forms instead of one? one for available locales and one for supported? 
It seems like deface overrade should insert it and it doesn't, I can't figure how to debug. 
tvdeyen, I'm available in slack if you'd like to move this conversation there. I'd like to merge this thing finally